aspnet/publishing/iis.rst

```diff
@@ -126,7 +132,7 @@ Troubleshooting:
 
 - If your application uses the `.UseUrls(...)` extension on `WebHostBuilder`, make sure you have positioned the `.UseUrls(...)` extension before the `.UseIISIntegration()` extension on `WebHostBuilder`. `.UseIISIntegration()` must overwrite any values you provide in `.UseUrls(...)` in order for the reverse-proxy to succeed.
 
-Issue #2
+Issue #3
```

Comment:
Can we replace all the #'s with a title - for example
Issue Browser no response?